styles/styles.css

```diff
@@ -0,0 +1,568 @@
+@import url("http://fonts.cdnfonts.com/css/cocogoose");
+
+html,
+body,
+header,
+nav {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: 'Lato', sans-serif;
+
+  /* font-family: 'COCOGOOSE', sans-serif; */
+  font-weight: 100;
+  word-wrap: break-word;
+}
+
+:root {
+  --black: #272a31;
+  --orange: #ec5242;
+  --gray: #d3d3d3;
+}
+
+ul {
+  margin: 0;
+  list-style-type: none;
+}
+
+.topnavbar {
+  background-color: var(--black);
+  color: whitesmoke;
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used around 20 times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```